Gemfile

```diff
@@ -60,10 +60,13 @@ group :development do
   gem 'listen', '~> 3.9'
   gem 'lookbook', '~> 2.3.4'
   gem 'rack-mini-profiler'
-  gem 'rubocop', '~> 1.66', require: false
-  gem 'rubocop-performance', '~> 1.25', require: false
-  gem 'rubocop-rails', '~> 2.26', require: false
-  gem 'rubocop-rspec', '~> 2.25', require: false
+  gem 'rubocop', require: false
+  gem 'rubocop-capybara', require: false
+  gem 'rubocop-factory_bot', require: false
+  gem 'rubocop-performance', require: false
+  gem 'rubocop-rails', require: false
+  gem 'rubocop-rspec', require: false
+  gem 'rubocop-rspec_rails', require: false
```

Comment:
We were requiring some of these (FactoryBot, Capybara, RSpec rails), but they appear to have been extracted to separate Gems as part of the Rubocop plugin changes. Looks like they were previously bundled in as part of the RSpec extension